2017/Day 2-2.py

```diff
@@ -0,0 +1,19 @@
+from itertools import combinations
+checksum = 0
+with open("Day 2 - input", "r") as file:
+    for line in file:
+        row = [int(i) for i in line.split('\t')]
+        combs = list(combinations(row,2))
+        for combination in combs:
+            result = 0
+            first = combination[0]/combination[1]
+            second = combination[1]/combination[0]
```

Comment:
No need to make a list
You can use the for loop on the combinations itself, like this
```
for comb in combinations(row, 2)
```
Also you can unwrap the combination and assign it to variables in the for loop line
```
for i, j in combinations(row, 2):
    first = i / j
    second = j / i
````